modules/protobuf/27.0/presubmit.yml

```diff
@@ -0,0 +1,67 @@
+matrix:
+  platform: ["debian11", "macos", "ubuntu2004"]
+  bazel: [6.x, 7.x]
+
+tasks:
+  verify_targets:
+    name: "Verify build targets"
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+    - '--host_cxxopt=-std=c++14'
+    - '--cxxopt=-std=c++14'
+    - '--host_copt=-Wno-deprecated-declarations'
+    - '--copt=-Wno-deprecated-declarations'
+    build_targets:
+    - '@protobuf//:protobuf'
+    - '@protobuf//:protobuf_lite'
+    - '@protobuf//:protoc'
+    - '@protobuf//:test_messages_proto2_cc_proto'
+    - '@protobuf//:test_messages_proto3_cc_proto'
+
+  verify_targets_windows:
+    name: "Verify build targets"
+    platform: windows
+    bazel: ${{ bazel }}
+    build_flags:
+    - '--host_cxxopt=-std=c++14'
+    - '--cxxopt=-std=c++14'
+    build_targets:
```

Comment:
Could we also add `@protobuf//python/...` to build test as a lot of people rely on these bindings?

Reply:
looks like both python/... and python:all will require more work upstream. https://buildkite.com/bazel/bcr-presubmit/builds/5850#018fc561-0b10-4a85-83d2-6b1b523e31b6

Reply:
IIUC this (or at least a subset of this) was working in v23 in BCR which I uploaded but we are blocked upgrading this dependency in our code base as this has regressed in newer versions.

Reply:
yea i think some things regressed upstream, similar to https://github.com/protocolbuffers/protobuf/pull/16884/

Reply:
i'd like to get this PR as a baseline and then we can start pulling in upstream patches like that one

Reply:
Makes sense

Reply:
The numpy pip deps should be a test only dep. Agree that adding @protobuf/python/... should be done separately in protobuf repo separate from this PR .

I wonder if increasing test coverage in bzlmod presubmits itself is right -- its tricky to have to make any patches in BCR after protobuf has already released.

Would it make sense to limit presubmits here to a sanity-check (e.g. avoid picking up spurious warnings) and add these to protobuf's CI to catch problems before releasing.

Reply:
Yea I think the long term ideal here is we add a WORKSPACE.bzlmod upstream, and then add an upstream CI job that uses bzlmod which should prove all that out. but right now since upstream is still loading the original WORKSPACE, it's hiding bzlmod issues and we're finding them here, and when users try to include it, instead